### PR TITLE
Adjust Nav Styling

### PR DESCRIPTION
**Pull request by joshri**
Closes: #1459

image

All you have to do to make a Nav Item gray is add the key/value `{sub: true}` to the navItems object array.

Also did some fixes for the logo, background, and border radius on main containers.

### CODE REVIEW

File: ui/components/Logo.tsx [hunk 1]
```diff
@@ -15,10 +15,10 @@ type Props = {
 function Logo({ className }: Props) {
   return (
     <Spacer padding="medium">
-      <Flex className={className}>
+      <Flex className={className} align>
         <img src={logoSrc} />
         <Spacer padding="xxs" />
-        <img src={titleSrc} />
+        <img src={titleSrc} style={{ height: "90%" }} />
```
Comment (jpellizzari) [inline ui/components/Logo.tsx:21]: What is this change trying to accomplish?

File: ui/pages/v2/Sources.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ function SourcesList({ className }: Props) {
       loading={isLoading}
       className={className}
     >
-      <SourcesTable sources={sources} />
+      <SourcesTable sources={sources || []} />
```
Comment (jpellizzari) [inline ui/pages/v2/Sources.tsx:20]: This looks like a leaky abstraction. Can we make `<SourcesTable />` handle `undefined` instead?

**Review by jpellizzari — COMMENTED** [review]
This doesn't look like the designs. We probably need to change the font of the sub menu item and make sure that menu items are grouped together.

File: ui/pages/v2/Sources.tsx (hunk 2 above)
Comment (joshri) [reply ui/pages/v2/Sources.tsx:20]: I'm going to remove this and handle it when I do the filtering. It has nothing to do with the Nav I was just checking out the errors

File: ui/components/Logo.tsx (hunk 1 above)
Comment (joshri) [reply ui/components/Logo.tsx:21]: Check it out without the set height - it stretches the text out to fit the container.

**Comment by joshri** [comment]
@jpellizzari good call out on the grouping together - I missed that. Sub menu item font I think needs to go from 600 - 400 font weight based on figma

**Comment by joshri** [comment]
image

FIXED

File: ui/components/Layout.tsx [hunk 3]
```diff
@@ -141,12 +144,26 @@ function Layout({ className, children }: Props) {
...
+                  <Flex column>
+                    <StyleLinkTab
+                      key={n.value}
+                      label={n.label}
+                      to={formatURL(n.value)}
+                      value={n.value}
+                    />
+                    {_.map(n.subItems, (sub) => {
+                      return (
+                        <StyleLinkTab
```
Comment (jpellizzari) [inline ui/components/Layout.tsx:156]: Looking at Figma, sub items should be `12px` below the main one. This looks a like a little bigger gap.

**Review by jpellizzari — APPROVED** [review]
Nice work on the sub items pattern. One nit about the distance, but assuming that matches the Figma drawing, LGTM.

**Review by jpellizzari — APPROVED** [review]

**Pull request merged**